HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -164,6 +170,13 @@ def customiseFor2017DtUnpacking(process):
 
     return process
 
+# Use 8p fit (PR29617) and don't apply HB- correction (PR26177)
+def customiseFor29617(process):
+    for producer in producers_by_type(process, "HBHEPhase1Reconstructor"):
+        producer.algorithm.chiSqSwitch = cms.double(-1.0)
+        producer.algorithm.applyLegacyHBMCorrection = cms.bool( False )
+    return process
+
```

Comment:
This subroutine should be called (see end of the file).

Reply:
Thanks for comment. It's now called in `customizeHLTforCMSSW`.